### PR TITLE
UI Test Updates & Drop In Legacy Removal

### PR DESCRIPTION
**Pull request by scannillo**
- Removed all references to Braintree Legacy Drop In.

- Added UI test for public PR #133. 

- Made UI tests expiration date dynamic.

### CODE REVIEW

File: BraintreeDropIn/BTCardFormViewController.m [hunk 1]
```diff
@@ -618,12 +618,10 @@ - (void)basicTokenization {
             self.navigationItem.rightBarButtonItem = addCardButton;
 
             if (error != nil) {
-                dispatch_async(dispatch_get_main_queue(), ^{
```
Comment (jackellenberger) [inline BraintreeDropIn/BTCardFormViewController.m]: Is this change related to drop in legacy?

Comment (scannillo) [reply BraintreeDropIn/BTCardFormViewController.m]: We already switched to the main thread at line 615 and didn't need to do it again.
